app/routes/create-catch-certificate/$catchCertificate/what-are-you-exporting/whatAreYouExporting.ts

```diff
@@ -145,4 +148,45 @@ export const getAddSpeciesLoaderData = async (catchCertificate: string, faoCode:
     stateCode,
     presentationCode
   }
+}
+
+export const addSpecies = async (catchCertificate: string | undefined, requestBody: any): Promise<any> => {
+  if(!catchCertificate) {
+    throw new Error("catchCertificate is required");
+  }
+
+  const response = await fetch(ADD_SPECIES_URL,
```

Comment:
this post is in more then one place and is quite verbose... does make sense to create a generic wrapper in common and just call it where we need?

Reply:
Updated @daniele-zurico to use a reusable function called `callApi: (url: string, requestHeaders?: HeadersInit,  method?: string, requestBody?: any) => Promise<Response>`